internal/commands/vaultsecrets/secrets/displayer.go

```diff
@@ -50,6 +53,15 @@ func (d *displayer) SetDefaultFormat(f format.Format) *displayer {
 	return d
 }
 
+func (d *displayer) SetSingleSecret(s string) *displayer {
```

Comment:
Maybe SetSecetType would be a better name here?

Reply:
Whoops, done in 4c4b9512b38617bf1145e21d06b988c4f8713289